src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,58 @@
+package pages;
+import org.apache.commons.logging.Log;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+
+        //locators
+
+//    pom
+//        By submitButtonLocator = By.cssSelector("[type='submit']");
+//        By emailFieldLocator = By.cssSelector("[type='email']");
+//        By passwordFieldLocator = By.cssSelector("[type='password']");
+
+//    page factory
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.